CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.5.0
+> Released on ??
+
+### Features
+
+* Add support for role name `Connector` for `konnect_team_role` and `konnect_system_account_role` resources
```

Comment:
@mheap IMO the changes in this PR aren't significant enough to be added to changelog (unless I'm missing something) - is it fine if I leave the changelog blank?

Reply:
Agree, I'd skip the changelog here